config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/InboundJmsDestinationDetail.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import static java.util.Arrays.stream;
+
+public class InboundJmsDestinationDetail extends JmsDestinationDetail {
+    
+    private AcknowledgeType acknowledgeType;
+    private ServiceResolutionSettings serviceResolutionSettings;
+    private String failureQueueName;
+    private boolean isEnabled;
+    private Integer numOfConsumerConnections;
+    private Integer maxMessageSizeBytes; // null = default, -1 = Do not override,  0 = Unlimited.
+    
+    public InboundJmsDestinationDetail() {
+    }
+    
+    public InboundJmsDestinationDetail(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 9 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWeegEhEh8YERCev6R1j&open=AWeegEhEh8YERCev6R1j)

[ISSUE_KEY]: AWeegEhEh8YERCev6R1j